src/openssl/ssl/context.cr

```diff
@@ -1,22 +1,40 @@
 class OpenSSL::SSL::Context
-  def self.default : self
-    @@default ||= new
-  end
-
   def initialize
     @handle = LibSSL.ssl_ctx_new(LibSSL.sslv23_method)
+    self.verify_mode = OpenSSL::SSL::VerifyMode::PEER
```

Comment:
Oh no, it is for the server validating the certificate of the client during TLS client authentication. I just wonder if setting it already makes it mandatory or prompts the client to provide a certificate. The doc is clear that it does give the client the opportunity to send a certificate if this is set.